### DISCUSSION TITLE
How to enable a relationship to display its label ?

### DISCUSSION THREAD

**Discussion by joylix** [neo4j-contrib/neovis.js#360] [Q&A] [answered]
How do I display the label of the relationship in the new version of Neovis', to say with simple_example.html,  In the older version, I knew that " caption: true " could be used, but I found that it didn't work in version 2.1.0.

**Comment by thebestnom** [comment, marked as answer]
Current way is to use advanced function and put 
```js
label: rel => rel.type
```
Or somthing on that end (I should really add helpers to the simple thingss 😅)

**Comment by joylix** [comment]
Thank you for your answer. Where can I find detailed help information on the usage of the new version of Neovis command?
